### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:Jaws and NVDA are not announcing appropriate information on changing the status of the checkboxes in "Settings  dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1524]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Tab to and select File->Settings.
3. In "settings" dialog , tab to and select plugin tree item.
4. Tab to list items under the table and check/uncheck .
5. observe the behaviour of Jaws and NVDA

**Actual Result:**
Jaws and NVDA are not announcing appropriate information on changing the status of the checkboxes

**Jaws Behaviour:**
Table
column 2 ,row 8
true

**NVDA Behaviour**
1  checked  selected  row 1  column 2
1  checked  selected  row 2  column 2

**Expected Result:**
Jaws and NVDA should announce the status of the checkboxes, ie checked/unchecked

Jaws and NVDA should announce as
Table
Column 2 ,Row 8
checked

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![settings-checkbox-explorer](https://user-images.githubusercontent.com/22956130/36189611-832b0596-1179-11e8-824f-49fe8b14462a.jpg)
![settings-checkbox-position](https://user-images.githubusercontent.com/22956130/36189618-8902d99e-1179-11e8-8630-edb72d3a6c77.jpg)
![settings-checkbox-explorer](https://user-images.githubusercontent.com/22956130/36189621-8d852f44-1179-11e8-9c81-00a2f2eebcae.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by Eskibear** [comment]
And it's out of the toolkit's scope.

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749971
